site/css/typography/_fonts.css

```diff
@@ -74,7 +74,12 @@
   font-size: 40px;
 }
 
-.fontL, .fontM, .fontStaticM {
+.fontL {
+  font-size: 30px;
+  line-height: 1.2em;
+}
```

Comment:
I updated `fontL` to use it in this pull request. The styleguide has change and this is the updated `fontL`. I'll be making a PR later aligning the rest of the fonts to the updated styleguide.